Marlin/src/pins/stm32f1/pins_MKS_ROBIN_E3_common.h

```diff
@@ -155,6 +155,17 @@
     #define DOGLCD_SCK                      PB13
     #define DOGLCD_MOSI                     PB15
 
+  #elif ENABLED(MKS_MINI_12864_V3)
+    #define ENABLE_SPI3
```

Comment:
So, theoretically, you could attach a FYSETC MINI 12864 v2.1 and it would also work?

Reply:
If that is the case, then this might as well just be `#elif ENABLED(FYSETC_MINI_12864_2_1)`.

Reply:
> So, theoretically, you could attach a FYSETC MINI 12864 v2.1 and it would also work?

// MKS MINI12864 and MKS LCD12864B; If using MKS LCD12864A (Need to remove RPK2 resistor)
  #if ENABLED(FYSETC_GENERIC_12864_1_1)

    #define LCD_BACKLIGHT_PIN               27
    #define LCD_RESET_PIN                   23
    #define DOGLCD_A0                       16
    #define DOGLCD_CS                       17
    #define DOGLCD_SCK                      PB13
    #define DOGLCD_MOSI                     PB15
    #define SDSS                            53
    #define BTN_EN1                           31
    #define BTN_EN2                           33
    #define BTN_ENC                           35
    #define BEEPER_PIN                        37
    #define SD_DETECT_PIN                     49

But I still have a blue screen

Reply:
> > So, theoretically, you could attach a FYSETC MINI 12864 v2.1 and it would also work?
> 
> // MKS MINI12864 and MKS LCD12864B; If using MKS LCD12864A (Need to remove RPK2 resistor)
> #if ENABLED(FYSETC_GENERIC_12864_1_1)
> 
> ```
> #define LCD_BACKLIGHT_PIN               27
> #define LCD_RESET_PIN                   23
> #define DOGLCD_A0                       16
> #define DOGLCD_CS                       17
> #define DOGLCD_SCK                      PB13
> #define DOGLCD_MOSI                     PB15
> #define SDSS                            53
> #define BTN_EN1                           31
> #define BTN_EN2                           33
> #define BTN_ENC                           35
> #define BEEPER_PIN                        37
> #define SD_DETECT_PIN                     49
> ```
> 
> But I still have a blue screen

You can try to add '#define FORCE_SOFT_SPI' in pin define, 
![image](https://user-images.githubusercontent.com/16629551/125009579-6428a180-e097-11eb-8262-23fc5cbe1edd.png)

Reply:
I have made some progress after  '#define FORCE_SOFT_SPI' now the blue screen is shown with a white rectangle...
![IMG_1149](https://user-images.githubusercontent.com/40673160/125053218-23d02000-e0a5-11eb-88f0-77f4d7e746d7.JPG)

Reply:
#if ENABLED(FYSETC_GENERIC_12864_1_1)

    #define DOGLCD_CS                       PA4
    #define DOGLCD_A0                       PA5
    #define LCD_PINS_DC                     DOGLCD_A0
    #define LCD_BACKLIGHT_PIN               -1
    #define LCD_RESET_PIN                   PA6
    #define NEOPIXEL_PIN                    PA7
    #define DOGLCD_MOSI                     PB15
    #define DOGLCD_SCK                      PB13
    #define FORCE_SOFT_SPI

All OK!!! THANKSSSSS

Reply:
The next challenge is to make the SD work, when I try to read the screen it freezes ...

Reply:
> The next challenge is to make the SD work, when I try to read the screen it freezes ...

I have the same problem in my test. 
@rhapsodyv 
![image](https://user-images.githubusercontent.com/16629551/125061758-3fa4e780-e0e0-11eb-9207-476b215cce02.png)
Any ideas? I think it is SPI influence, because PA7 (Neopixel pin) is SPI1, and reader is SPI2

Reply:
The SD is detected on the screen it shows the correct message but nevertheless when you enter it remains blocked, I think that if the SD we put it in MKS it does the same.